### PR TITLE
[config/show] add support for vlan configuration

### PR DESCRIPTION
**Pull request by ishidawataru**
https://github.com/Azure/sonic-py-swsssdk/pull/22 and https://github.com/Azure/sonic-py-swsssdk/pull/23 need to be get merged before this PR.

```
$ config vlan add 1000
$ config vlan member add 1000 Ethernet1
$ config vlan member add 1000 Ethernet9_1 -u
$ config vlan add 2000
$ config vlan member add 2000 Ethernet1
$ config vlan member add 2000 Ethernet9_2 -u

$ show vlan config
Name        VID  Member       Mode
--------  -----  -----------  --------
Vlan1000   1000  Ethernet1    tagged
                 Ethernet9_1  untagged
Vlan2000   2000  Ethernet1    tagged
                 Ethernet9_2  untagged

$ config vlan member del 1000 Ethernet1
$ config vlan member del 2000 Ethernet1

$ show vlan config
Name        VID  Member       Mode
--------  -----  -----------  --------
Vlan1000   1000  Ethernet9_1  untagged
Vlan2000   2000  Ethernet9_2  untagged

$ config vlan del 1000
$ config vlan del 2000

$ show vlan config
Name    VID    Member    Mode
------  -----  --------  ------
```

### CODE REVIEW

**Comment by lguohan** [comment]
both swsssdk PR has been merge, can you help to resolve the conflict?

**Comment by ishidawataru** [comment]
@lguohan done.

File: config/main.py [hunk 1]
```diff
@@ -189,6 +189,94 @@ def load_minigraph():
...
+        print "{} already exists".format(vlan)
+        raise click.Abort
+    db.set_entry('VLAN', vlan, {'vlanid': vid})
+
+@vlan.command('del')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.pass_context
+def del_vlan(ctx, vid):
+    db = ctx.obj['db']
+    db.set_entry('VLAN', 'Vlan{}'.format(vid), None)
```
Comment (yxieca) [inline config/main.py]: Would it be better to delete vlan members before deleting the vlan itself?

Comment (ishidawataru) [reply config/main.py]: Thanks for your review. fixed.

**Pull request merged**